.github/workflows/go-generate-without-stringer.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env sh
+#
+# This is a temporary hack to work around
+# https://github.com/golang/go/issues/51629 , wherein the stringer
+# generator doesn't work with generics.
+#
+# This script is the equivalent of `go generate ./...`, except that it
+# only runs generate on packages that don't try to use stringer.
+
+find . -name '*.go' | xargs grep -l go:generate | xargs -n1 dirname | sort -u | while read dir; do
+	if ! grep cmd/stringer $dir/*.go; then
+		set -x
```

Comment:
Is this intended to be equivalent to `go generate || true`?

Reply:
Sort of. My intent is to skip go generate tasks that we know are going to fail, while still reporting failures for others. What I appear to have discovered, is that this workflow never complained if `go generate` fails (vs. succeeds but produces a diff), oops. Working on it.